examples/bundle-browserify/README.md

```diff
@@ -15,3 +15,8 @@ You should see the following:
 
 ![](https://ipfs.io/ipfs/QmNtpcWCEd6LjdPNfBFDaVZdD4jpgT8ZTAwoFJXKhYMJdo/1.png)
 ![](https://ipfs.io/ipfs/QmNtpcWCEd6LjdPNfBFDaVZdD4jpgT8ZTAwoFJXKhYMJdo/2.png)
+
+## Special note
+
+In order to use js-ipfs in the browser, you need to replace the default `zlib` library by `browserify-zlib-next`, a full implementation of the native `zlib` package, full in Node.js. 
+See the package.json to learn how to do this and avoid this pitfall. More context on: https://github.com/ipfs/js-ipfs#use-in-the-browser-with-browserify-webpack-or-any-bundler
```

Comment:
"see the package.json" for what? I'm not sure but probably you mean "see the webpack.config.js" for aliasing zlib to browserify-zlib-next no?